Ginger/Ginger/Actions/UCValueExpression.xaml.cs

```diff
@@ -132,6 +132,7 @@ private void OpenExpressionEditorButton_Click(object sender, RoutedEventArgs e)
         {
             ValueExpressionEditorPage w = new ValueExpressionEditorPage(obj, AttrName, HideVEBusinessFlowAndActivityVariables);
             w.ShowAsWindow();
+            ValueTextBox.Text = Convert.ToString(obj.GetType().GetProperty(AttrName).GetValue(obj));
```

Comment:
why we still have this ValueTextBox.Text = here? why it is needed?

Reply:
Whatever we select it is not updating the text on action page, there NotifyPropertyChange is not implemented so it is not updating the action page, on the other places where the "ValueExpressionEditorPage" page is called the same implementation is done, so please let me know if i should implement the "NotifyPropertyChange" this will take some time since it needs to be handled through reflection.